frontend/views/components/Modal/Modal.vue

```diff
@@ -86,25 +93,26 @@ export default {
       }
       this.updateUrl()
     },
-    unloadModal (name) {
+    unloadModal () {
       if (this.subcontent.length) {
         this.subcontent.pop()
       } else {
         this.content = null
+        // Refocus on the window focusable element (mainly hack for esc button)
+        document.querySelectorAll('button, a, input, select, textarea, [tabindex]:not([tabindex="-1"])')[0].focus()
```

Comment:
This isn't A11Y correct... Here we are selecting the first focusable element in the doc.

What we should do is to focus the element that triggered the modal in the first place (so the user continues where he left on). So, when the modal is created, we should store the currently focused element (ex: the button saying "Login") and then when we close it (in this line), we focus that element again.